docs/microsoft-365-developer-program-get-started.md

```diff
@@ -1,18 +1,25 @@
 ---
-title: Set up a Microsoft 365 developer sandbox subscription
+title: Request a Microsoft 365 developer sandbox subscription
 description: Set up a Microsoft 365 developer subscription for building solutions independent of your production environment.
 ms.date: 04/01/2019
 ms.localizationpriority: high
 ---
 
-# Set up a Microsoft 365 developer sandbox subscription 
+# Request a Microsoft 365 developer sandbox subscription 
 
-Set up a Microsoft 365 developer sandbox to build your solutions independent of your production environment. Your developer program membership, which you join as a regular member or through Visual Studio enterprise, includes a Microsoft 365 E5 developer sandbox subscription with 25 user licenses. It lasts for 90 days and is free to use for development purposes (coding solutions) only. Commercial transactions, including purchasing paid services, are not supported.
+Qualifying Microsoft 365 Developer Program members can set up a Microsoft 365 E5 developer subscription for use in building solutions independent of your production environment. The Microsoft 365 E5 developer sandbox subscription includes 25 user licenses and lasts for 90 days. If you use your subscription for valid development activity, it will renew automatically. Commercial transactions, including purchasing paid services, are not currently supported.
 
 > [!NOTE] 
-> To set up a subscription, you must first [join the Microsoft 365 Developer Program](microsoft-365-developer-program.md) directly or through Visual Studio Professional or Enterprise (if you're a subscriber). After joining, you'll see the option to set up a subscription.
+> Before you request a developer subscription, you must first [join the Microsoft 365 Developer Program](microsoft-365-developer-program.md) directly or through Visual Studio Professional or Enterprise (if you're a subscriber). 
 
-You can choose to set up either an instant sandbox or a configurable sandbox.
+If you qualify for a Microsoft 365 E5 subscription through the developer program, the subscription set up flow will start automatically after you join the program. If you choose not to set up your subscription right away, the option to set up a subscription will be available on your [Microsoft 365 Developer Program dashboard](https://developer.microsoft.com/en-us/microsoft-365/profile).
```

Comment:
This is not accurate if we are removing the configurable sandbox. Recommend to remove.

Reply:
For now we aren't removing the configurable sandbox option but we will need to update this later when we are able to do that.